doc/whats_new/v1.2.rst

```diff
@@ -84,6 +84,10 @@ Changelog
 :mod:`sklearn.preprocessing`
 ............................
 
+- |Fix| Fixes a bug in :class:`preprocessing.OneHotEncoder` such that
+  `drop_idx_` properly references the dropped category in the `categories_`
+  attribute when there are infrequent categories. :pr:`25589` by `Thomas Fan`_.
```

Comment:
Nit: how about this active form phrasing?
```suggestion
- |Fix| :meth:`preprocessing.OneHotEncoder.drop_idx_` now properly 
  references the dropped category in the `categories_` attribute
  when there are infrequent categories. :pr:`25589` by `Thomas Fan`_.
```

Reply:
Instead of `:meth:`, it should be `:attr:` I THINK.

Reply:
Thanks! I did not know that this directive existed.